pandas/lib.pyx

```diff
@@ -26,9 +26,9 @@ from cpython cimport (PyDict_New, PyDict_GetItem, PyDict_SetItem,
                       PyBytes_GET_SIZE,
                       PyUnicode_GET_SIZE)
 
-try:
+IF PY_MAJOR_VERSION == 2:
```

Comment:
I tried that. It does not work for me. Running ``python3 setup.py cython`` results in a ``lib.c`` with PyString_GET_SIZE in the translated ``max_len_string_array``, where if it worked that would become ``PyUnicode_GET_SIZE``